### PR TITLE
Did some optimization which makes transfer time usage shorter

### PR DESCRIPTION
**Pull request by wqj97**
time usage shorten to One-third

### CODE REVIEW

**Comment by freekmurze** [comment]
Because of all the styling changes it's hard to see what your actually change is.

The instanciation of the `Imagick` class seems to be move up to the constructor where it's being set as an instance variable.

Could you please remove all other changes?

File: src/Pdf.php [hunk 1]
```diff
@@ -29,6 +33,7 @@ public function __construct($pdfFile)
             throw new PdfDoesNotExist();
         }
 
+        $this->image_obj = new \Imagick($pdfFile);
```
Comment (freekmurze) [inline src/Pdf.php]: Please rename `image_obj` to `imagick`

Comment (wqj97) [inline src/Pdf.php]: ok ~

**Comment by wqj97** [comment]
when I using $pdf->setsetPage(1)->saveImage()
I find it's too slow.
Then, I try to new Imagick() separately,
It seems instanciation spent lots of time.

**Pull request closed without merging**